packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -81,6 +81,11 @@ export interface IRuntimeStateHandler {
 	isAttached: () => boolean;
 }
 
+function isEmptyBatchPendingMessage(message: IPendingMessageFromStash): boolean {
```

Comment:
Ah, dang, I didn't think about how localOpMetadata is dropped on stash.  Good fix.